sidebars/index.js

```diff
@@ -0,0 +1,12 @@
+function requireUncached(module) {
+  delete require.cache[require.resolve(module)];
+  return require(module);
+}
+
+const Home = requireUncached('./home');
+const Core = requireUncached('./core');
```

Comment:
This was weird. We have HMR, and on refresh, these files were being rerun, but they weren't being updated and rerun, just the old version was being rerun. I would have assumed that editing getFilesToWatch in the docs plugin lifecycle (or defining a new plugin to do so) would have been sufficient, but I guess not. I mean, the files are rerunning, but they're being cached they're not reflecting future changes.

I wonder if the docusaurus people would have anything to say about this. Maybe it just has to be like this, but I suspect not...

Reply:
Let's remove this and just know that HMR doesn't work for this